cryostat/templates/deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
           - name: GRAFANA_DATASOURCE_URL
             value: http://127.0.0.1:8080
           - name: GRAFANA_DASHBOARD_URL
-            value: "{{ if .Values.grafana.ingress.enabled }}http{{ if .Values.grafana.ingress.tls }}s{{ end }}://{{ with index .Values.grafana.ingress.hosts 0 }}{{ .host }}{{ end }}{{ end }}"
+            value: "{{ if .Values.grafana.ingress.enabled }}http{{ if (or (.Values.grafana.ingress.tls) (.Values.core.sslProxied)) }}s{{ end }}://{{ with index .Values.grafana.ingress.hosts 0 }}{{ .host }}{{ end }}{{ end }}"
```

Comment:
I'm not sure why someone would do this, but it's possible that the Grafana service could be exposed in a different way than the core service (e.g. an Ingress with no TLS). To cover this case, how about also adding a `grafana.sslProxied` as well? This way the core and grafana configurations can be kept independent.

Reply:
Hi @ebaron, I agree with comment and I've updated my PR accordingly.

Thank you for your comments!